api/src/main/java/gov/va/vro/api/model/AbdBloodPressure.java

```diff
@@ -1,11 +1,15 @@
 package gov.va.vro.api.model;
```

Comment:
These files look like duplication of the same files in service-data-access. I guess the change here probably won't affect the others.